AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -277,38 +277,35 @@ - (void)testLogsFlushedImmediatelyWhenIntervalIsOver {
 }
 
 - (void)testLogsNotFlushedImmediatelyWhenIntervalIsCustom {
-
+  
```

Comment:
This PR is adding whitespaces, please run the clang formatter and remove them

Reply:
Well, I there is a comment on a next line; As for whitespaces - that Is what I get from clang-formatter